semantic-python/src/Language/Python/Core.hs

```diff
@@ -278,9 +290,17 @@ instance Compile Py.IfStatement where
 instance Compile Py.ImportFromStatement
 instance Compile Py.ImportStatement
 instance Compile Py.Integer
+
+-- Buggy: ignores the semantics of the provided name. This requires
+-- coordination between Call nodes and KeywordArgument nodes, which is
+-- a sign that we need something smarter here, but it'll do for now.
```

Comment:
I think it suggests we shouldn’t have a `Compile` instance for `Py.KeywordArgument`, and should instead pattern match on it in the `Py.Call` instance.